### PR TITLE
Minor changes:

### PR DESCRIPTION
**Pull request by SuperMairio**
- css colour changes to make webpage easier to read + make hyperlinks more obvious
- Added hint to challenge 6
- Added linebreak to CLI errors
- Made use wget/curl error clearer
- Removed confusing large error
- Hidden answers in README behind collapsable span
- Edited hint hint to make it clearer
- Minor grammar alterations

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -53,35 +53,10 @@
                 "Please see the following link http://nameofwebsite/help",
                 "Task failed successfully",
                 "Task successfully failed",
-                "big error, much problem",
-                """
-<html><body>
```
Comment (yeroc-sebrof) [inline app.py:58]: I liked this error 😢

File: app.py [hunk 2]
```diff
@@ -108,7 +83,7 @@ def challenge5():
     if (os.getenv("PORT") == '8080' and re.search('(curl|wget)\/*', request.headers.get('User-Agent').lower())):
         return "Challenge String: Vjgfqgug it a qugwvy dopo vrql"
     else:
-        return pick_one(not_right)
+        return pick_one(not_right) #leaving this one as newline not required for webpage
```
Comment (yeroc-sebrof) [inline app.py]: What do you mean by newline is not requred for webpage? This is challenge 5

File: app.py [hunk 3]
```diff
@@ -118,21 +93,21 @@ def challenge5hint1():
         else:
             return "Have a look into common TCP ports"
     else:
-        return "Are you using Curl or wget? I don't think you are"
+        return "I wouldn't look for hints on a cURL challenge using the browser..." # Making it more obvious that user should curl for hints
```
Comment (yeroc-sebrof) [inline app.py:98]: I like this better than what I put

File: app.py [hunk 4]
```diff
@@ -53,35 +53,10 @@
...
-   915                      A: 00000000/00000000   B: 00000000/00000000         <br>
-   915                      C: 00000000/00000000   D: 00000000/7F6B5050         <br>
-   915                      E: 00000000/84806000   F: 00000000/00000004         <br>
-   915              END OF SYMPTOM DUMP    <br>
-</body></html>
-                """
+                "big error, much problem"
             ]
 
+errormsg = not_right + "\n" # adding newline otherwise terminal looks messy
```
Comment (yeroc-sebrof) [inline app.py]: This line won't work. You're appending a new line to the end of the array.

File: app.py (hunk 2 above)
Comment (SuperMairio) [reply app.py]: oops sorry got confused, thought that was something else. Please delete comment.

File: app.py (hunk 1 above)
Comment (SuperMairio) [reply app.py:58]: Was confusing, especially if someone is new to cURL and might think that is a correct output

**Review by yeroc-sebrof — APPROVED** [review]
Thank you 😄

File: app.py (hunk 4 above)
Comment (SuperMairio) [reply app.py]: I noticed this and it should be fixed in my latest commit :)

**Pull request merged**